src/rinterface_extra.c

```diff
@@ -3146,7 +3146,7 @@ SEXP R_igraph_incidence(SEXP incidence, SEXP directed, SEXP mode, SEXP multiple)
   c_mode = (igraph_neimode_t) Rf_asInteger(mode);
   c_multiple=LOGICAL(multiple)[0];
                                         /* Call igraph */
-  IGRAPH_R_CHECK(igraph_incidence(&c_graph, &c_types, &c_incidence, c_directed, c_mode, c_multiple));
+  IGRAPH_R_CHECK(igraph_biadjacency(&c_graph, &c_types, &c_incidence, c_directed, c_mode, c_multiple));
```

Comment:
I would suggest also renaming `R_igraph_incidence()` for consistency. The R functions have already been renamed, see #900.